core/store/src/trie/shard_tries.rs

```diff
@@ -216,7 +216,6 @@ impl ShardTries {
         self.0.flat_storage_manager.clone()
     }
 
-    // TODO (#9989): Change visibility to crate once we remove state_snapshot_every_n_blocks from chain
```

Comment:
Is it no longer relevant?

Reply:
Yes, I realized we can't remove the functionality of "when to snapshot" from chain as we need epoch manager; which means we would need to expose the snapshot config from trie.